src/main/java/ip/Duke.java

```diff
@@ -0,0 +1,287 @@
+                    description = strList[0];
+                    extraInfo = strList[1];
+                    newTask = new Event(description, extraInfo);
+                    break;
+                default:
+                    throw new DukeException("Type of task is not recognised!");
+            }
+            taskList.add(newTask);
+            String singulStr = "Now you have 1 task in the list";
+            if (taskList.size() == 1) {
+                prettyPrint(String.format("%s\n%s   %s\n%s %s",
+                        CREATE_MESSAGE, TAB, newTask.toString(), TAB, singulStr));
+            } else {
+                prettyPrint(String.format("%s\n%s   %s\n%s Now you have %d tasks in the list.",
+                        CREATE_MESSAGE, TAB, newTask.toString(), TAB, taskList.size()));
+            }
+        } catch (DukeException e) {
+            prettyPrint(e.toString());
+        }
+    }
+
+    /**
+     * Marks a task at index as done
+     */
+    private static void markTask(String index) {
+        index = index.strip();
+        int i = Integer.parseInt(index);
+        i--;
+        Task task = taskList.get(i);
+        task.markAsDone();
+        prettyPrint(String.format("%s\n%s   %s", MARK_MESSAGE, TAB, task.toString()));
+    }
+
+    /**
+     * Marks a task at index as not done
+     */
+    private static void unmarkTask(String index) {
+        index = index.strip();
+        int i = Integer.parseInt(index);
+        i--;
+        Task task = taskList.get(i);
+        task.markAsUndone();
+        prettyPrint(String.format("%s\n%s   %s", UNMARK_MESSAGE, TAB, task.toString()));
+    }
+
+    private static void deleteTask(String index) {
+        index = index.strip();
+        int i = Integer.parseInt(index);
+        i--;
+        Task task = taskList.remove(i);
+        String singulStr = "Now you have 1 task in the list";
+        if (taskList.size() == 1) {
+            prettyPrint(String.format("%s\n%s   %s\n%s %s",
+                    DELETE_MESSAGE, TAB, task.toString(), TAB, singulStr));
+        } else {
+            prettyPrint(String.format("%s\n%s   %s\n%s Now you have %d tasks in the list.",
+                    DELETE_MESSAGE, TAB, task.toString(), TAB, taskList.size()));
+        }
+    }
+
+    /**
+     * Creates a new Collection that has each entry numbered in ascending order
+     */
+    private static void printAll() {
+        List<String> printables = new ArrayList<>();
+        printables.add(LIST_MESSAGE);
+        for (int i = 0; i < taskList.size(); i++) {
+            Task task = taskList.get(i);
+            int index = i + 1;
+            printables.add(String.format("%d.%s", index, task.toString()));
+        }
+        prettyPrint(printables);
+    }
+
+    private static void prettyPrint() {
+        prettyPrint("");
+    }
+
+    private static void prettyPrint(String printable) {
+        String out = String.format("%s\n%s %s\n%s", LINE, TAB, printable, LINE);
```

Comment:
Perhaps using a noun for variable names would be better able to capture its value/meaning? I noticed the same issue in several other places too